unittests/backend/cassandra/BackendTests.cpp

```diff
@@ -89,29 +89,6 @@ TEST_F(BackendCassandraTest, Basic)
             "3E2232B33EF57CECAC2816E3122816E31A0A00F8377CD95DFA484CFAE282656A58"
             "CE5AA29652EFFD80AC59CD91416E4E13DBBE";
 
-        auto hexStringToBinaryString = [](auto const& hex) {
```

Comment:
Removing duplicates and introducing some StringUtils that can be used from multiple places.